CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Dependencies are updated to the latest available version during each release. Th
 - All commands that took a `--settings` option to specify the path to the configuration file now take a `--config-path` option instead. This name is clearer and avoids introducing a separate "settings" term.
 - The default path to the Gafaelfawr configuration file is now taken from the `GAFAELFAWR_CONFIG_PATH` environment variable rather than `GAFAELFAWR_SETTINGS_PATH`, for the same reason.
 
+### New features
+
+- The response from the `/auth` now includes any `Authorization` or `Cookie` headers from the incoming request with Gafaelfawr tokens and secrets filtered out. This is automatically used by `GafaelfawrIngress` resources to filter those secrets out of the request passed to the protected service so that the user's credentials are not leaked to services. Manual ingress configurations should add `Authorization` and `Cookie` to the `nginx.ingress.kubernetes.io/auth-response-headers` annotation.
```

Comment:
Maybe instead of phrasing as "now includes ... filtered out" it can be rephrased as "now excludes..."

Reply:
> Manual ingress configurations should add `Authorization` and `Cookie` to the `nginx.ingress.kubernetes.io/auth-response-headers` annotation.

"Should add" in order to what? In order to filter these headers? Or for the necessary functioning of Gafaelfawr?